contracts/strategies/MobiusStrategy.sol

```diff
@@ -284,12 +325,14 @@ contract MobiusStrategy is Ownable, IStrategy {
         uint256 amount = 0;
         uint256 additionalAmount = 0;
         if (!disableRewardTokenClaim) {
-            amount = gauge.claimable_reward(address(this), address(celo));
-            additionalAmount = gauge.claimable_tokens(address(this));
+            if (address(gauge) != address(0)) {
+                amount = gauge.claimable_reward(address(this), address(celo));
+                additionalAmount = gauge.claimable_tokens(address(this));
```

Comment:
nothing wrong here, just double checking - is this where we get the balance of MOBI tokens?

Reply:
yes

Reply:
OK, got it. here also, for dev readability, I'd just add some dev comments mentioning what each method does (one removes celo rewards and the other, mobi rewards).